blog-posts/angular-inject-the-injector.md

```diff
@@ -0,0 +1,252 @@
+---
+path: "/blog/angular-inject-the-injector"
+date: "2021-06-08"
+title: "The \"Inject the Injector\" pattern"
+featuredImage: images/georgios-kaleadis-aBTfTMsOCOI-unsplash.jpg 
+imageCredits: We want to credit people for their work
```

Comment:
should this actually list people to credit for images?

Reply:
yes, then I switched plans and just took my own image but actually I would like to ship this feature with this PR so I try to push a change along the text update. Thank for the hint!